home-work-app/webpack.config.dev.js

```diff
@@ -0,0 +1,35 @@
+const path = require('path');
+const webpack = require('webpack');
+const ExtractTextPlugin = require("extract-text-webpack-plugin");
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+const CaseSensitivePathPlugins = require('case-sensitive-paths-webpack-plugin');
+
+module.exports = {
+    mode: 'development',
+
+    devtool: 'source-map',
+
+    output: {
+        filename: '[name].js',
+        path: path.join(__dirname, 'public', 'dev')
+    },
+
+    
+    plugins: [
+        new CaseSensitivePathPlugins(),
+        new webpack.DefinePlugin({
+            BROWSER_SUPPORTS_HTML5: true
+        }),
+        new webpack.ProvidePlugin({ // common imports for all the source
+            react: 'React',
+            _: 'lodash'
+        }),
+        new HtmlWebpackPlugin({
+            hash: true,
+            template: './index.html'
+        }),
+        new ExtractTextPlugin('styles.css'),
+    ],
+
+    watch: true
```

Comment:
Let's remove it.
I suppose you don't need to rebuild the app on each change

Reply:
done